### PR TITLE
DOC-2411: TinyMCE 7.1.1 Release Notes and Changelog.

### PR DESCRIPTION
**Pull request by FarzadHayat**
Ticket: DOC-2411

Site: [Staging branch](http://docs-feature-711-doc-2411.staging.tiny.cloud/docs/tinymce/latest/7.1.1-release-notes)

Changes:
* DOC-2411: 7.1.1 Release Notes template outline

Pre-checks:
- [x] Branch prefixed with `feature/<version>/`, `hotfix/<version>/`, `staging/<version>/`, or `release/<version>/`
- [x] `modules/ROOT/nav.adoc` has been updated `(if applicable)`
- [x] Files has been included where required `(if applicable)`
- [x] Files removed have been deleted, not just excluded from the build `(if applicable)`
- [x] Files added for `New product features`, and included a `release note` entry.
- [x] Fix merge conflict in `release-notes.adoc` after TinyMCE 7.1 is released.
- [x] Fix merge conflict in `nav.adoc` after TinyMCE 7.1 is released.
- [x] Update `changelog.adoc` to include 7.1.1 entries.
- [x] Re-update `changelog.adoc` after missing tickets have been backported.

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

File: modules/ROOT/pages/7.1.1-release-notes.adoc
Comment (spocke) [inline modules/ROOT/pages/7.1.1-release-notes.adoc:33]: We decided to move this to 7.2

File: modules/ROOT/pages/changelog.adoc
Comment (spocke) [inline modules/ROOT/pages/changelog.adoc:22]: I assume that this was a mistake that we made on the previous 7.1 release?

**Review by spocke — APPROVED** [review]

File: modules/ROOT/pages/7.1.1-release-notes.adoc
Comment (kemister85) [reply modules/ROOT/pages/7.1.1-release-notes.adoc]: All boilerplate content is removed before we merge back into the target release/7.1.1 branch.

File: modules/ROOT/pages/7.1.1-release-notes.adoc
Comment (kemister85) [reply modules/ROOT/pages/7.1.1-release-notes.adoc:33]: When was the target version changed? Happy to remove if we need it to.

File: modules/ROOT/pages/changelog.adoc
Comment (kemister85) [reply modules/ROOT/pages/changelog.adoc:22]: yeah, just a `asciidoc` vs `md` syntax error.

File: modules/ROOT/pages/7.1.1-release-notes.adoc
Comment (ShiridiGandham) [reply modules/ROOT/pages/7.1.1-release-notes.adoc:33]: @kemister85 Please remove this entry. It is now scheduled to go out with 7.2.

File: modules/ROOT/pages/7.1.1-release-notes.adoc
Comment (kemister85) [inline modules/ROOT/pages/7.1.1-release-notes.adoc:49]: Removed as this was moved to `7.2`.
```suggestion
```

File: modules/ROOT/pages/7.1.1-release-notes.adoc
Comment (FarzadHayat) [reply modules/ROOT/pages/7.1.1-release-notes.adoc]: Done ✅

Comment (kemister85) [reply modules/ROOT/pages/7.1.1-release-notes.adoc]: Also re-arranged the entries to be inline with the changelog.adoc @FarzadHayat

**Review by kemister85 — APPROVED** [review]

**Comment by FarzadHayat** [comment]
@ShiridiGandham need your approval before merging

**Review by ShiridiGandham — APPROVED** [review]
LGTM

**Pull request merged**
